sound/soc/intel/boards/sof_sdw.c

```diff
@@ -325,6 +325,14 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					RT711_JD2 |
 					SOF_SDW_FOUR_SPK),
 	},
+	/* MeteorLake devices */
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_PRODUCT_FAMILY, "Intel_mtlrvp"),
+		},
+		.driver_data = (void *)(RT711_JD1 | SOF_SDW_PCH_DMIC),
```

Comment:
Also add SOF_SDW_TGL_HDMI, even if we don't have HDMI working now it will be usable when the topology adds HDMI.

Reply:
Ack